README.md

```diff
@@ -25,12 +25,81 @@ Quality assurance framework for Magnetic Resonance Imaging
 ## Overview
 
 ---
+hazen is a software framework for performing automated analysis of MRI Quality Assurance data.
 
+Some examples of what hazen can perform:
 
+| SNR measurements   | Spatial Resolution measurements |
+| ------------------ | ------------------------------- |
+| _insert snr image_ | _insert spatial res image_      |
```

Comment:
@tomaroberts can you add something here to just remove the examples section?

Reply:
These were placeholders to remind me for when I overhaul the README.md

Reply:
@laurencejackson @hshuaib90 

I've temporarily made a separate branch for the README edits so as not to interfere with this PR. You can view latest version here: https://github.com/GSTT-CSC/hazen/tree/readme-dev

Also includes re-jigging of usage instructions, i.e.: Docker first, then Mac/Linux, then M1 users.

Reply:
Tested on python 3.9.7 on intel mac and everything works as expected following the readme you linked